stylesheets/_component.daterange.scss

```diff
@@ -1,204 +1,298 @@
+    font-size: 14px;
+    position: absolute;
+    left: 8px;
+    top: 8px;
+}
+
+.daterangepicker .input-mini {
+    border: 1px solid color(gray, lightest);
+    border-radius: 4px;
+    color: color(gray, darker);
+    font-size: 14px;
+    height: 30px;
+    line-height: 30px;
+    display: block;
+    vertical-align: middle;
+    margin: 0 0 5px;
+    padding: 0 6px 0 28px;
+    width: 100%;
 }
 
 .daterangepicker td,
 .daterangepicker th {
-  text-align: center;
-  width: 20px;
-  height: 20px;
-  -webkit-border-radius: 4px;
-  -moz-border-radius: 4px;
-  border-radius: 4px;
-  cursor: pointer;
-  white-space: nowrap;
+    cursor: pointer;
+    height: 20px;
+    text-align: center;
+    white-space: nowrap;
+    width: 20px;
+}
+
+.daterangepicker .table-condensed > tbody > tr > td,
+.daterangepicker .table-condensed > tbody > tr > th,
+.daterangepicker .table-condensed > tfoot > tr > td,
+.daterangepicker .table-condensed > tfoot > tr > th,
+.daterangepicker .table-condensed > thead > tr > td,
+.daterangepicker .table-condensed > thead > tr > th {
+    padding: 5px;
+}
+
+.daterangepicker .calendar th,
+.daterangepicker .calendar td {
+    min-width: 32px;
+    text-align: center;
+    white-space: nowrap;
+}
+
+.daterangepicker td.in-range {
+    background: #ebf4f8;
+    border-radius: 0;
+}
+
+.daterangepicker td.start-date {
+    border-radius: 4px 0 0 4px;
+}
+
+.daterangepicker td.end-date {
+    border-radius: 0 4px 4px 0;
 }
 
 .daterangepicker td.off {
-  color: #999;
+    color: color(gray, light);
 }
 
 .daterangepicker td.disabled {
-  color: #999;
+    color: color(gray, light);
 }
 
 .daterangepicker td.available:hover,
 .daterangepicker th.available:hover {
-  background: #eee;
-}
-
-.daterangepicker td.in-range {
-  background: #ebf4f8;
-  -webkit-border-radius: 0;
-  -moz-border-radius: 0;
-  border-radius: 0;
+    background: color(gray, off-white);
 }
 
 .daterangepicker td.active,
 .daterangepicker td.active:hover {
-  background-color: #357ebd;
-  border-color: #3071a9;
-  color: #fff;
+    background-color: color(primary);
+    color: color(white);
 }
 
 .daterangepicker td.week,
 .daterangepicker th.week {
-  font-size: 80%;
-  color: #ccc;
+    color: color(gray, lightest);
+    font-size: 80%;
 }
 
 .daterangepicker select.monthselect,
 .daterangepicker select.yearselect {
-  font-size: 12px;
-  padding: 1px;
-  height: auto;
-  margin: 0;
-  cursor: default;
+    cursor: default;
+    font-size: 12px;
+    height: auto;
+    margin: 0;
+    padding: 1px;
 }
 
 .daterangepicker select.monthselect {
-  margin-right: 2%;
-  width: 56%;
+    margin-right: 2%;
+    width: 56%;
 }
 
 .daterangepicker select.yearselect {
-  width: 40%;
+    width: 40%;
 }
 
 .daterangepicker select.hourselect,
 .daterangepicker select.minuteselect,
 .daterangepicker select.ampmselect {
-  width: 50px;
-  margin-bottom: 0;
+    margin-bottom: 0;
+    width: 50px;
+}
+
+// scss-lint:disable PseudoElement
+.daterangepicker:before,
```

Comment:
Definitely need single semicolon pseudos here? I've disabled the linter for it but thought best to check